### ISSUE TITLE
CTRL-c not effective. 

### ISSUE THREAD

**Issue by jarpy** [elastic/kibana-docker#88]
When running Kibana with an attached terminal like:

```
docker run --rm -it docker.elastic.co/kibana/kibana:6.3.1
```

the process does not respond to CTRL-c.

Nor does it respond to `SIGINT` or `SIGTERM` sent via the `kill` command.

cc @tbragin

**Comment by tbragin** [comment]
Looks like @jbudz tried to fix this before, but we need another go at it: https://github.com/elastic/kibana/pull/15896

**Comment by tianon** [comment]
See https://github.com/docker-library/kibana/pull/32 for how we fixed it -- we just added `tini` (which from an upstream perspective is just a hacky workaround).

IMO, a _much better_ fix would be to reopen/reconsider https://github.com/elastic/kibana/pull/15896 and fix it properly in Kibana itself rather than punting via `tini`.

In the meantime, folks using Docker can get around this issue via `docker run --init` or `init: true` in their YAML (which tells Docker to inject `tini` automatically).
